public/content/roadmap/pectra/7702/index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Pectra 77O2 guidelines
+description: Learn more about 7702 in the Pectra release
+lang: en
+---
+
+## Abstract
```

Comment:
the headings are missing the heading-id to work with the sidebar. Pushing a commit in a few to fix this